themes/rjorgenson/rjorgenson.theme.sh

```diff
@@ -3,99 +3,103 @@
 
 # set colors for use throughout the prompt
 # i like things consistent
-BRACKET_COLOR=${blue}
-STRING_COLOR=${green}
+OMB_THEME_BRACKET_COLOR="${OMB_THEME_BRACKET_COLOR:-${blue}}"
+OMB_THEME_STRING_COLOR="${OMB_THEME_STRING_COLOR:-${green}}"
 
 SCM_THEME_PROMPT_PREFIX=""
 SCM_THEME_PROMPT_SUFFIX=""
 
 SCM_THEME_PROMPT_DIRTY=" ${bold_red}✗${normal}"
 SCM_THEME_PROMPT_CLEAN=" ${bold_green}✓${normal}"
-SCM_GIT_CHAR="${STRING_COLOR}±${normal}"
+SCM_GIT_CHAR="${OMB_THEME_STRING_COLOR}±${normal}"
 SCM_SVN_CHAR="${bold_cyan}⑆${normal}"
 SCM_HG_CHAR="${bold_red}☿${normal}"
 
+PROMPT_CHAR="${OMB_THEME_BRACKET_COLOR}➞ ${normal}"
+if [[ $OSTYPE == *darwin* ]]; then
+    PROMPT_CHAR="${OMB_THEME_BRACKET_COLOR}➞  ${normal}"
+fi
+
 #Mysql Prompt
 export MYSQL_PS1="(\u@\h) [\d]> "
 
+TITLEBAR=""
 case $TERM in
-        xterm*)
-        TITLEBAR="\[\033]0;\w\007\]"
-        ;;
-        *)
-        TITLEBAR=""
-        ;;
+        xterm*) TITLEBAR="\[\033]0;\w\007\]" ;;
 esac
 
-PS3=">> "
 
 __my_rvm_ruby_version() {
-    local gemset=$(echo $GEM_HOME | awk -F'@' '{print $2}')
-  [ "$gemset" != "" ] && gemset="@$gemset"
-    local version=$(echo $MY_RUBY_HOME | awk -F'-' '{print $2}')
-    local full="$version$gemset"
-  [ "$full" != "" ] && echo "${BRACKET_COLOR}[${STRING_COLOR}$full${BRACKET_COLOR}]${normal}"
+    local gemset ; gemset=$(echo "${GEM_HOME}" | awk -F'@' '{print $2}')
+    local version ; version=$(echo "${MY_RUBY_HOME}" | awk -F'-' '{print $2}')
+    [ "${gemset}" != "" ] && gemset="@${gemset}"
+    local full="${version}${gemset}"
+    [ "${full}" != "" ] \
+      && echo "${OMB_THEME_BRACKET_COLOR}[${OMB_THEME_STRING_COLOR}${full}${OMB_THEME_BRACKET_COLOR}]${normal}"
 }
 
 is_vim_shell() {
-        if [ ! -z "$VIMRUNTIME" ]
-        then
-                echo "${BRACKET_COLOR}[${STRING_COLOR}vim shell${BRACKET_COLOR}]${normal}"
-        fi
+    if [ -n "${VIMRUNTIME}" ] ; then
+        echo "${OMB_THEME_BRACKET_COLOR}[${OMB_THEME_STRING_COLOR}vim shell${OMB_THEME_BRACKET_COLOR}]${normal}"
+    fi
+}
+
+is_integer() { # helper function for todo-txt-count
+    [ "$1" -eq "$1" ] > /dev/null 2>&1
+        return $?
 }
 
-function is_integer() { # helper function for todo-txt-count
+is_integer() { # helper function to make sure input is an integer
     [ "$1" -eq "$1" ] > /dev/null 2>&1
         return $?
 }
```

Comment:
Ah, we now actually have double definitions of `is_integer`. I'll remove it at my side.